### DISCUSSION TITLE
Next Gen images - Web p doesnt appear to be served

### DISCUSSION THREAD

**Discussion by WCL-joejones** [breezefront/community#237] [1. Help]
Hi Guys,

we are tying to implement the next-gen image config from [here](https://breezefront.com/docs/next-gen-images#configure-nginx)

we have added in the nginx config and they look to be serving the correct url just the non webp version despite there being a .jpg.webp version available and accessible

we have tried:
flushing magento cache
flushing magento image cache
flushing varnish
nothing appears in the nginx error log

Any help would be greatly appreciated 

Kind Regards
Joe

**Comment by vovayatsyuk** [comment]
Sorry, but I don't know how to help you. 

Are you sure your Nginx uses the same config file you edit?
Are you sure that images are served using Nginx?
